src/status.ts

```diff
@@ -1,23 +1,21 @@
 import { threadId } from 'worker_threads'
 
-export type StatusMethod = ((emoji: string, message: string) => void) | ((message: string) => void)
+export type StatusMethod = (icon: string, ...message: string[]) => void
 
 export interface Status {
     info: StatusMethod
     error: StatusMethod
 }
 
-function transform(emojiOrMessage: string, message?: string): string {
-    return `[${threadId ? threadId.toString().padStart(4, '_') : 'MAIN'}] ${[emojiOrMessage, message]
-        .filter(Boolean)
-        .join(' ')}`
+function getPrefix(): string {
+    return `[${threadId ? threadId.toString().padStart(4, '_') : 'MAIN'}]`
 }
 
 export const status: Status = {
-    info(emojiOrMessage: string, message?: string) {
-        console.info(transform(emojiOrMessage, message))
+    info(icon: string, ...message: any[]) {
+        console.info(getPrefix(), icon, ...message.filter(Boolean))
     },
-    error(emojiOrMessage: string, message?: string) {
-        console.error(transform(emojiOrMessage, message))
+    error(icon: string, ...message: any[]) {
+        console.error(getPrefix(), icon, ...message.filter(Boolean))
```

Comment:
As it is now and looking at the usage, why not just remove the `icon` argument?

Reply:
It's just for verbosity really.

Reply:
It's a bit confusing if not used as intended. The entire message is said to be an icon...
![image](https://user-images.githubusercontent.com/53387/105366328-815bd780-5bff-11eb-90e2-888fd30bcee7.png)

Reply:
That's a few oversights, fixed. :)